extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -575,9 +585,19 @@ export class AmpA4A extends AMP.BaseElement {
           const extensions = extensionsFor(this.win);
           creativeMetaDataDef.customElementExtensions.forEach(
             extensionId => extensions.loadExtension(extensionId));
+          // Make sure we don't collapse this slot.
+          creativeMetaDataDef.collapse = false;
           return creativeMetaDataDef;
         })
         .catch(error => {
+          if (error == NO_CONTENT_RESPONSE) {
+            return {
+              minifiedCreative: null,
```

Comment:
Yeah...  I'm not sure how much I trust the type checker here.  :-/  It seems to have a low false positive rate, but a high false negative rate.  Thanks for the change.